assets/javascript/game.js

```diff
@@ -0,0 +1,194 @@
+//Load when the document is ready
+$(document).ready(function () {
+
+    //Create the array of desingers
+    var desingers = [
+        "saul bass",
```

Comment:
If you are using words that are using 2 words instead of one, people may not know that they need to use the space bar